xml/soc-operations-gpu-passthrough.xml

```diff
@@ -8,9 +8,8 @@
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="gpu-passthrough">
  <title>GPU passthrough</title>
  <para>
-  &productname; supports PCI passthrough (PCIPT) for better network
-  performance, improving network I/O, decreasing latency, and reducing
-  processor overhead.
+  &productname; GPU passthrough functionality provides the nova instance direct access to the GPU device for a
+    better performance.
```

Comment:
See Jeremy's comment earlier.

s/a better performance/better performance/

Reply:
Updated in the latest commit